### DISCUSSION TITLE
Event loading screen

### DISCUSSION THREAD

**Discussion by Sakura-ne** [DzunN86/SecondHand#55] [Q&A] [answered]
Simple way implement loading page / screen ?
like Loading when load product data

![enter image description here](https://repository-images.githubusercontent.com/182145367/0ef7ee00-62db-11e9-9f5c-ac6a166a3bac)

**Comment by DzunN86** [comment, marked as answer]
> Simple way implement loading page / screen ? Ex: Loading when load product data
We can implement it in every component that necessarily to be loaded in loading data. and utilize state isLoading for condisional rendering compoent like skeleton
```javascript
if (isLoading) {
    return (
      <View style={styles.cardProduct}>
        <Skeleton animation="wave" style={styles.imageProduk} />
        <Skeleton
          animation="wave"
          width={60}
          style={{
            marginTop: 15,
            borderRadius: RADIUS.small,
          }}
        />
        <Skeleton
          animation="wave"
          width={100}
          style={{
            marginTop: 10,
            borderRadius: RADIUS.small,
          }}
        />
        <Skeleton
          animation="wave"
          width={100}
          style={{
            marginTop: 10,
            borderRadius: RADIUS.small,
          }}
        />
      </View>
    );
  }

return (
// Screen / component
)
```
by using Skeleton component It will give good experince than we use `<ActivityIndicator/>` component
